packages/@coorpacademy-components/src/molecule/teams-popin/style.css

```
@@ -0,0 +1,102 @@
+@value colors: "../../variables/colors.css";
+@value dark from colors;
+@value brand from colors;
+
+
+.disable {
+  top: 0px;
+  position: relative;
+  width: 100%;
+  margin: 0;
+  padding: 15% 25%;
+  background-color: '#eceff1';
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  overflow: hidden;
+  height: 100%;
+}
+
+.popin {
+  top: 30%;
+  position: absolute;
+  width: 35%;
+  overflow: auto;
+  background: white;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-shadow: 0 0 15px 0 rgba(166, 166, 166, 0.84);
+  border-radius: 8px;
+  padding: 20px;
+}
+
+.header {
+  font-family: 'Open Sans';
+  font-size: 24px;
+  font-weight: bold;
+  color: '#13171a';
```

`@value black from colors;`
```suggestion
  color: black;
```